openmls/src/group/public_group/diff/apply_proposals.rs

```diff
@@ -0,0 +1,164 @@
+use std::collections::HashSet;
```

Comment:
If only we could see moved code ... I'll assume that most of the code in public_group was moved and only look at the API.

Reply:
Yes. The only difference is that instead of deriving the external pub keypair internally, it returns the external proposal s.t. the caller can do it.